doc/Reference/Markup/Converters.md

```diff
@@ -0,0 +1,119 @@
+---
+uid: Reference.Markup.Converters
+---
+# Converters
+
+Converters are components that allow for the conversion of values from one type to another during C# markup processing.
+Converters are particularly useful when you want to display data from a domain model or data source in a customized way, applying formatting, calculations, or transformations before displaying it.
+
+## Using a Converter
+
+There may be times when you may want to simply use a converter instead of providing a delegate over and over. For these scenarios you may want to provide some sort of Static readonly context like:
```

Comment:
```suggestion
There may be times when you may want to simply use a converter instead of providing a delegate over and over. For these scenarios, you may want to provide some sort of `static readonly` context like:
```